package.json

```diff
@@ -67,6 +70,7 @@
     "esbuild": "0.14.39",
     "eslint": "8.11.0",
     "events": "3.3.0",
+    "istanbul-badges-readme": "1.8.1",
```

Comment:
this can be a dev dependency yeah?

Reply:
Sanity: this is already a dev dependency right?